java/http/java-http-consumer/src/main/java/HttpConsumer.java

```diff
@@ -83,12 +104,21 @@ public HttpConsumer(HttpConsumerConfig config, CountDownLatch messagesReceivedLa
                 OpenTelemetry.noop().getTracer("client-examples") :
                 GlobalOpenTelemetry.getTracer("client-examples");
     }
-
+    @SuppressWarnings("ST_WRITE_TO_STATIC_FROM_INSTANCE_METHOD")
```

Comment:
As well you don't need this one (in general you don't need both suppress warnings and an exclusion in the XML file)

Reply:
Removed